non-msrv-tests/tests/ui/item.stderr

```diff
@@ -1,53 +1,51 @@
-error: no traits found to implement, use `#[derive_where(..)` to specify some
- --> tests/ui/item.rs:6:1
+error: empty `derive_where` found
+ --> tests/ui/item.rs:7:1
```

Comment:
This message changed for some reason

Reply:
Yes, there a test case that needs to be removed actually.

Reply:
See https://github.com/ModProg/derive-where/pull/33/files#r780830161.